models/public/index.md

```diff
@@ -198,6 +198,7 @@ Image inpainting task is to estimate suitable pixel information to fill holes in
 | Model Name                | Implementation | OMZ Model Name | Accuracy | GFlops | mParams |
 | ------------------------- | ---------------| -------------- | -------- | ------ | ------- |
 | GMCNN Inpainting          | TensorFlow\*   | [gmcnn-places2-tf](./gmcnn-places2-tf/README.md) | 33.47Db | 691.1589 | 12.7773|
+| Hybrid-CS-Model-MRI       | TensorFlow\*   | [hybrid_cs_model_mri](./hybrid_cs_model_mri/README.md) | 00.00Db | | |
```

Comment:
If it's in the inpainting table, shouldn't the `task_type` also be set to `image_inpainting`?